04-gas/gas-section-two/gas-section-two.css

```diff
@@ -1,52 +1,107 @@
+.gas-section-two-subcontainer,
+.gas-section-two-subcontainer,
+.gas-section-two-copy-container,
+.gas-section-two-title-container,
+.gas-section-two-paragraph-container,
+.gas-section-two-title,
+.gas-section-two-paragraph{
+  border: var(--temporary-border);
+}
+
 .gas-section-two-container {
   display: grid;
   grid-template-columns: 1fr;
+  padding: var(--container-padding-mobile);
+  background-color: var( --level1-background-color);
+}
+
+@media (min-width:600px){
+  .gas-section-two-container {
+    padding: var(--container-padding-desktop);
+  }
 }
 
 .gas-section-two-subcontainer {
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: var(--subcontainer-padding-mobile);
+  background-color: var( --level2-background-color);
+}
+
+@media (min-width:600px){
+  .gas-section-two-container {
+    padding: var(--subcontainer-padding-desktop);
+  }
+}
+
+.gas-section-two-copy-container {
   display: grid;
   grid-template-columns: 1fr;
   grid-template-areas: 
-    "gas-section-two-title-container"
-    "gas-section-two-paragraph";
-  justify-content: center;
-  align-content: center;
-  row-gap: var(--items-gap-mobile);
-  padding: var(--container-padding-mobile);
+  "gas-section-two-title-container"
+  "gas-section-two-paragraph-container";
+  align-content: start;
+  row-gap: var(--subcontainer-row-gap-mobile);/*in this poin i add same row for subcontainer because of design/var(--copy-container-row-gap-mobile);*/
```

Comment:
can you explain this a bit?
this is the gap between the "2" and the "Only turn on the heating when you need it"
do you want to make a new global variable?
or you want to use an existing one? 
the problem is if at the end we change something on all of them, and we don't remember this has a specific case.. it might get forgotten..
or maybe i misunderstand you

Reply:
both of this global css are exist, but as this section doesn't have images i thought its better to use same one which we use on subcontainer i will remember it and i need to ask Karleen about that.